backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/format/ImpFormat.java

```diff
@@ -54,6 +61,8 @@ private ImpFormat(
 		this.manualImport = manualImport;
 		this.importTableDescriptor = importTableDescriptor;
 		this.columns = ImmutableList.copyOf(columns);
+		this.charset = charset;
+		this.skipFirstNRows = skipFirstNRows;
```

Comment:
pls validate/normalize this parameter
e.g.
skipFirstNRows = skipFirstNRows > 0 ? skipFirstNRows : 0;

Reply:
as in "don't allow negative values".

Done